.github/workflows/build.yml

```diff
@@ -12,32 +12,34 @@ jobs:
 
     runs-on: ${{ matrix.os }}
 
+    defaults:
+      run:
+        shell: bash -l {0}
+
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        mpi: [nompi, openmpi]
+        mpi: [nompi, mpich]
+        libxc: [6, 7]
         config: [Debug]
-        version: [13]
+        exclude:
```

Comment:
Possibly needs a comment as to why.

Reply:
It is just to reduce $\mathrm{CO_2}$ emissions, as the serial binary is already tested in combination with libxc 7.

Reply:
```suggestion
        # Serial binary is tested with libxc 7
        exclude:
```

Reply:
Incorporated.